test/runtests.jl

```diff
@@ -2115,6 +2115,18 @@ using Aqua: Aqua
             series = bseries(AverageVectorFieldMethod(BigFloat), 7)
             # TODO: This test is currently broken and throws an error
             @test_broken is_energy_preserving(series)
+
+            # Generated from MatLab
```

Comment:
This does not really describe anything that helps people reading this code. Could you please describe how you generated it and what kind of properties it has?